src/core/components/divider/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { css, SerializedStyles } from '@emotion/react';
+
+import { border, text } from '@guardian/src-foundations/palette';
+import { space } from '@guardian/src-foundations';
+import { textSans } from '@guardian/src-foundations/typography';
+
+type Props = {
+	size?: 'full' | 'partial';
+	spaceAbove?: 'tight' | 'loose';
+	displayText?: string;
+};
+
+const fullStyles = css`
+	margin-left: -10px;
+	width: calc(100% + 20px);
+`;
+const partialStyles = css`
+	margin-left: -10px;
+	width: 160px;
+`;
```

Comment:
Please can we move the styles to a separate `styles.ts` file in the `divider` directory?